community/tools/ra-data-hasura/src/index.js

```diff
@@ -32,6 +30,32 @@ export default (serverEndpoint, headers) => {
       throw new Error(JSON.stringify({'error': 'Invalid table/schema resource'}));
     }
 
+    return {schema: schema, tableName: tableName};
```

Comment:
You can trim this down to:

```suggestion
    return { schema, tableName };
```

This is due to https://alligator.io/js/object-property-shorthand-es6/ in ES6.